hack/e2e-node-test.sh

```diff
@@ -14,121 +14,27 @@
-  if [[ $zone == "" ]]; then
-    echo "Could not find gcloud compute/zone when running:\ngcloud info --format='value(config.properties.compute.zone)'"
-    exit 1
-  fi
-
-  # Get the compute project
-  project=$(gcloud info --format='value(config.project)')
-  if [[ $project == "" ]]; then
-    echo "Could not find gcloud project when running:\ngcloud info --format='value(config.project)'"
-    exit 1
-  fi
-
-  # Check if any of the images specified already have running instances.  If so reuse those instances
-  # by moving the IMAGE to a HOST
-  if [[ $images != "" ]]; then
-  IFS=',' read -ra IM <<< "$images"
-       images=""
-       for i in "${IM[@]}"; do
-         if [[ $(gcloud compute instances list "${instance_prefix}-$i" | grep $i) ]]; then
-           if [[ $hosts != "" ]]; then
-             hosts="$hosts,"
-           fi
-           echo "Reusing host ${instance_prefix}-$i"
-           hosts="${hosts}${instance_prefix}-${i}"
-         else
-           if [[ $images != "" ]]; then
-             images="$images,"
-           fi
-           images="$images$i"
-         fi
-       done
-  fi
-
-  # Parse the flags to pass to ginkgo
-  ginkgoflags=""
-  if [[ $focus != "" ]]; then
-     ginkgoflags="$ginkgoflags -focus=$focus "
-  fi
-
-  if [[ $skip != "" ]]; then
-     ginkgoflags="$ginkgoflags -skip=$skip "
-  fi
-
-  if [[ $run_until_failure != "" ]]; then
-     ginkgoflags="$ginkgoflags -untilItFails=$run_until_failure "
-  fi
-
-  # Output the configuration we will try to run
-  echo "Running tests remotely using"
-  echo "Project: $project"
-  echo "Image Project: $image_project"
-  echo "Compute/Zone: $zone"
-  echo "Images: $images"
-  echo "Hosts: $hosts"
-  echo "Ginkgo Flags: $ginkgoflags"
-
-  # Invoke the runner
-  go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=2 --ssh-env="gce" \
-    --zone="$zone" --project="$project"  \
-    --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
-    --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
-    --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
-    --delete-instances="$delete_instances"
-  exit $?
-
-else
-  # Refresh sudo credentials if not running on GCE.
-  if ! ping -c 1 -q metadata.google.internal &> /dev/null; then
-    sudo -v || exit 1
-  fi
-
-  # Test using the host the script was run on
-  # Provided for backwards compatibility
-  "${ginkgo}" --focus=$focus --skip=$skip "${KUBE_ROOT}/test/e2e_node/" --report-dir=${report} \
-    -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true
-  exit $?
+if [[ -n "${SKIP:-}" ]]; then
+    ARGHELP="${ARGHELP}SKIP='${SKIP}'"
 fi
+
+echo "NOTE: $0 has been replaced by 'make test-e2e-node'"
+echo
+echo "The equivalent of this invocation is: "
+echo "    make test-e2e-node ${ARGHELP}"
+echo
+echo
+make --no-print-directory -C "${KUBE_ROOT}" test-e2e-node FOCUS=${FOCUS:-} SKIP=${SKIP:-}
```

Comment:
There were some other arguments/envvars that could have been passed.
I guess you are consciously ignoring them here and in ARGHELP. Can you please add a comment about i?